src/app/pages/registration/register-header/register-header.component.html

```diff
@@ -0,0 +1,10 @@
+  <div class="container">
+      <div class="image-container">
+        <img class="landvile-logo" src="../../../../assets//img/images/ICON/Logo.png"/>
```

Comment:
Ensure that you use absolute imports throughout

Reply:
I need help with this @kelvinndmo... This is an import of an image.